openmc/data/library.py

```diff
@@ -78,3 +94,38 @@ def export_to_xml(self, path='cross_sections.xml'):
         tree = ET.ElementTree(root)
         tree.write(path, xml_declaration=True, encoding='utf-8',
                    method='xml')
+
+    @classmethod
+    def from_xml(cls, path):
+        """Read cross section data library from an XML file.
+
+        Parameters
+        ----------
+        path : str
+            Path to XML file to read.
+
+        Returns
+        -------
+        data : openmc.data.DataLibrary
+            openmc.data.DataLibrary object initialized from the provided XML
```

Comment:
I prefer the simpler "Data library object initialized ..." since the type is already clear.